src/App.jsx

```diff
@@ -1,14 +1,71 @@
 import './App.css';
+import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import ColorChoice from './components/ColorChoice';
+import { useState } from 'react';
+
+// count like feature
+const calculateTotalLikes = (messages) =>
+  messages.reduce((total, message) => total + (message.liked ? 1 : 0), 0);
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
```

Comment:
Nice work on using the `useState` hook to initialize state for your `App` component.